### DISCUSSION TITLE
Add buttom to change the language

### DISCUSSION THREAD

**Discussion by JuniverseCoder** [massCodeIO/massCode#264] [Ideas]
In the version 3.4.0，there is no way to change the language of Snippets or Fragments. It will be nice to add buttom to change the language because some snippets/fragments will be different language.
![image](https://user-images.githubusercontent.com/16131756/187056335-d5078350-4f05-4c32-b689-89ce09d16e41.png)

**Comment by antonreshetov** [comment]
The fragment is no different from the snippet, and the language choice for it was there from the beginning. Just look at the bottom of the editor. If you were able to select the language for the snippet, you will also be able to select the language for the fragment.

**Reply by JuniverseCoder** [reply]
Thanks, I found the selector from the buttom of the editor.
